book/creating-a-basic-website/adding-a-template.rst

```diff
@@ -153,6 +153,32 @@ some more parameters, as for the `article`-property in the example above. The
 example is enabling the godMode, the icon for adding links and the icon for
 adding tables.
 
+Using XInclude
+-----------------
+A template can be composed of various snippets using `XInclude`_. To use Xinclude you first have to add the proper namespace (xmlns:xi="http://www.w3.org/2001/XInclude"-attribute).
```

Comment:
dont use snippets here because snippets has already a different meaning in sulu.